pkg/network/tcp_peer.go

```diff
@@ -210,7 +210,9 @@ func (p *TCPPeer) handleQueues() {
 	var p2pSkipCounter uint32
 	const p2pSkipDivisor = 4
 
-	var writeTimeout = p.server.TimePerBlock
+	// TODO: @roman-khimov, this one is controversial a bit, because time per block may be changed during the node's lifetime.
+	// Do we need to keep writeTimeout aligned with these changes?
```

Comment:
It's retrieved once per peer lifecycle and the value is cached in the contract, so I think it's OK to use it this way.

One unrelated caveat though is that this can be too aggressive for 100ms blocks, some lower bound is needed.

Reply:
Fixed.